ArduCopter/Parameters.h

```diff
@@ -381,12 +381,14 @@ class Parameters {
         k_param_vehicle = 257, // vehicle common block of parameters
         k_param_throw_altitude_min,
         k_param_throw_altitude_max,
-
+        k_param_land_accel_limit,       // 260
         // the k_param_* space is 9-bits in size
         // 511: reserved
+
     };
 
     AP_Int16        format_version;
+    AP_Int8        land_accel_limit;
```

Comment:
I think we should make this parameter an AP_Float and make the units m/s/s

Reply:
position controller takes via set_max_speed_accel_xy() method in cm/s/s. And I have seen good results in 10 cm/s/s after dropping from 100 cm/s/s [ 1m/s/s] . That is why I taken unit in cm/s/s. Can use AP_Float instead of AP_Int8.